docs/apis/avalanchego/apis/p-chain.md

```diff
@@ -993,15 +1010,16 @@ platform.getCurrentValidators({
   - `txID` is the validator transaction.
   - `startTime` is the Unix time when the validator starts validating the Subnet.
   - `endTime` is the Unix time when the validator stops validating the Subnet.
-  - `stakeAmount` is the amount of nAVAX this validator staked. Omitted if `subnetID` is not the Primary Network.
+  - `stakeAmount` is the amount of nAVAX this validator staked. Omitted if `subnetID` is not a PoS subnet.
   - `nodeID` is the validator’s node ID.
-  - `weight` is the validator’s weight when sampling validators. Omitted if `subnetID` is the Primary Network.
-  - `rewardOwner` is an `OutputOwners` output which includes `locktime`, `threshold` and array of `addresses`. Omitted if `subnetID` is not the Primary Network.
-  - `potentialReward` is the potential reward earned from staking. Omitted if `subnetID` is not the Primary Network.
+  - `weight` is the validator’s weight when sampling validators. Omitted if `subnetID` is a PoS subnet.
+  - `validationRewardOwner` is an `OutputOwners` output which includes `locktime`, `threshold` and array of `addresses`. Specifies the owner of the potential reward earned from staking. Omitted if `subnetID` is not a PoS subnet.
+  - `delegationRewardOwner` is an `OutputOwners` output which includes `locktime`, `threshold` and array of `addresses`. Specifies the owner of the potential reward earned from delegations. Omitted if `subnetID` is not a PoS subnet.
+  - `potentialReward` is the potential reward earned from staking. Omitted if `subnetID` is not a PoS subnet.
   - `delegationFeeRate` is the percent fee this validator charges when others delegate stake to them. Omitted if `subnetID` is not the Primary Network.
-  - `uptime` is the % of time the queried node has reported the peer as online. Omitted if `subnetID` is not the Primary Network.
+  - `uptime` is the % of time the queried node has reported the peer as online. Omitted if `subnetID` is not a PoS subnet.
```

Comment:
`subnetID is not a PoS subnet`   should be changed to  `subnetID is not that of a PoS subnet`  to be grammatically correct?

or `subnetID is not for a PoS subnet`

Reply:
I'm fine with `subnetID is not a PoS subnet` tbh